src/asm-manual.adoc

```diff
@@ -784,6 +784,10 @@ as seen by `objdump`:
       1c: R_RISCV_PCREL_LO12_S  .L0
 ----
 
+On RV32, `ld <rd>, <symbol>` and `sd <rd>, <symbol>, <rt>` can be used to
```

Comment:
Is <symbol>+<offset> allowed?

(Allowed in gas and llvm but the llvm AsmParser is a bit of a hack.)

Reply:
LLVM looks like it is passing the offset through to the relocation correctly, so I'm happy to allow this. I can clarify the wording to ensure it is, because there are other places that allow `<symbol>{+<offset>}` when they only say `<symbol>` right now.

Reply:
I updated this to be `<symbol expression>` which can be symbol, offset, or symbol with offset.